backend/rest/controller/tx.go

```diff
@@ -63,6 +64,12 @@ func registerQueryTxList(r *mux.Router) error {
 		if status != "" {
 			query["status"] = status
 		}
+		if address != "" {
+			query["$or"] = []bson.M{
```

Comment:
optimize suggestion：only `Transfer` tx query address by `from` or `to`, other tx type query address by `from`